client/src/components/BottomMenu.tsx

```diff
@@ -16,31 +17,73 @@ const NavLinkWithSearch = (props: React.ComponentProps<typeof NavLink>) => {
   );
 };
 
+const Menu = styled.div`
+  margin-right: calc(100% - 100vw);
+  position: fixed;
+  z-index: 2;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 2.5rem;
+  background-color: #343434;
```

Comment:
even though the actual color #343434 was added to the colors list in theme.tsx, let's also make it be used from the `theme.tsx`.

We need to interpolate the value.

Classical interpolation would be this:

```suggestion
  background-color: ${"#343434"};
```

But this, again is not pulled from the theme. It's still hardcoded. Interpolated but hardcoded.

Slightly better:



```suggestion
  //top of file
  import { theme } from "src/consts/theme";

  background-color: ${ theme.menu.backgroundColor };
```

But, this solution might not be totally reactive. When user changes the theme (dark to light), the value will not change.

And how it says in the docs to do it:

```suggestion
  background-color: ${({ theme }) => theme.menu.backgroundColor };
```

What is found inside the {} brackets is an arrow function:

```tsx
({ theme }) => theme.menu.backgroundColor
```

it's a shorthand for

```tsx
(props) => props.theme.menu.backgroundColor
```

whose return value is the color. The return value will change when the theme gets switched. Therefore, this is "reactive".